### ISSUE TITLE
seneca.js fatal 404

### ISSUE THREAD

**Issue by mcdonnelldean** [senecajs/senecajs.org#236]
Via: https://github.com/senecajs/seneca/issues/456

While receiving fatal error seneca print this message:
ALL ERRORS FATAL: action called with argument fatal$:true (probably a plugin init error, or using a plugin seneca instance, see senecajs.org/fatal.html)
at ....

but senecajs.org/fatal.html doesn't exists on the server (404)

**Comment by Tapppi** [comment]
This is still a problem..

**Comment by bra1n** [comment]
Still broken.

**Comment by nfantone** [comment]
@mcdonnelldean Do you have an idea as to where that link should point to? I could open a PR with a fix.

**Comment by mcdonnelldean** [comment]
I've had a look and don't see this page being there before. I'd say it's best to remove the link for now or to write a small page on it and add it in.

**Comment by bra1n** [comment]
It would be great if there *was* something about Fatal Errors there, because the general documentation seems to be rather lacking when it comes to this topic. :-)

**Comment by mcdonnelldean** [comment]
@bra1n I don't disagree. Right now most folk are at peak bandwidth. If anyone wants to take a stab at it, please go ahead.

**Comment by nfantone** [comment]
@bra1n Yeah, I agree. But since I don't really know what the original intention was, my vote is on removing the URL from the logs. We'll add it back when we have a working page for that.

PR will be up in a minute.

**Comment by nfantone** [comment]
@mcdonnelldean Removed the URL for now. See https://github.com/senecajs/seneca/pull/578
